modules/starfish/2.0/config/default.yaml

```diff
@@ -11,9 +11,8 @@ lcr-modules:
             # Each key:value pair is the name of the variant caller and the path to its VCF file
             # Each key should be unique in this set. 
             vcf: 
-                strelka: "results/gambl/strelka-1.1/99-outputs/vcf/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.strelka.combined.vcf.gz"
-                lofreq: "results/gambl/lofreq-1.0/99-outputs/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.lofreq.snvs.vcf.gz"
```

Comment:
I kinda liked it with all of these listed but if they go in the demo that's even better.